packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-requirements-array.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  OpenApi3_1Element,
+  SecurityRequirementElement,
+  SecuritySchemeElement,
+  isSecuritySchemeElement,
+  isComponentsElement,
+} from '@swagger-api/apidom-ns-openapi-3-1';
+import { AnnotationElement, toValue, isObjectElement, Element } from '@swagger-api/apidom-core';
+
+import type { Toolbox } from '../toolbox';
+
+type SecurityRequirementsArrayPluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const securityRequirementsArrayRefractorPlugin =
```

Comment:
We're just adding some explanation.

```suggestion
/*
 * OpenAPI 3.0.3
 * If the security scheme is of type "oauth2" or "openIdConnect", then the value is a list of scope names required for the execution, and the list MAY be empty if authorization does not require a specified scope. For other security scheme types, the array MUST be empty.
 *
 * OpenAPI 3.1.0
 * If the security scheme is of type "oauth2" or "openIdConnect", then the value is a list of scope names required for the execution, and the list MAY be empty if authorization does not require a specified scope. For other security scheme types, the array MAY contain a list of role names which are required for the execution, but are not otherwise defined or exchanged in-band.
 * This issue is conveying the fact that all other types than oauth2 or openIdConnect must have empty array [] as a value. In OpenAPI 3.1.0 the value MAY contain roles for other types.
 */
const securityRequirementsEmptyRolesRefractorPlugin =
```